src/core/public/chrome/ui/header/collapsible_nav.tsx

```diff
@@ -202,172 +207,214 @@ export function CollapsibleNav({
+                    const { ...hydratedLink } = createWorkspaceNavLink(href, workspace, navLinks);
+                    return {
+                      href,
+                      ...hydratedLink,
+                      'data-test-subj': 'collapsibleNavAppLink--workspace',
+                      onClick: async (event) => {
+                        if (!isModifiedOrPrevented(event)) {
+                          closeNav();
+                        }
+                      },
+                    };
+                  })}
+                  maxWidth="none"
+                  color="subdued"
+                  gutterSize="none"
+                  size="s"
+                />
+              ) : (
+                <EuiText size="s" color="subdued" style={{ padding: '0 8px 8px' }}>
+                  <p>
+                    {i18n.translate('core.ui.EmptyWorkspaceList', {
+                      defaultMessage: 'No Workspaces',
+                    })}
+                  </p>
+                </EuiText>
+              )}
+              <EuiText size="s" color="subdued" style={{ padding: '0 8px 8px' }}>
+                <p>
+                  {i18n.translate('core.ui.SeeMoreWorkspace', {
+                    defaultMessage: 'SEE MORE',
+                  })}
+                </p>
+              </EuiText>
+            </EuiCollapsibleNavGroup>
+            <EuiCollapsibleNavGroup onClick={closeNav} iconType={'managementApp'} title={'Admin'} />
+          </>
+        )}
 
-              return {
-                ...hydratedLink,
-                'data-test-subj': 'collapsibleNavAppLink--recent',
-                onClick: (event) => {
-                  if (!isModifiedOrPrevented(event)) {
-                    closeNav();
-                    onClick(event);
-                  }
-                },
-              };
-            })}
-            maxWidth="none"
-            color="subdued"
-            gutterSize="none"
-            size="s"
-            className="osdCollapsibleNav__recentsListGroup"
-          />
-        ) : (
-          <EuiText size="s" color="subdued" style={{ padding: '0 8px 8px' }}>
-            <p>
-              {i18n.translate('core.ui.EmptyRecentlyViewed', {
-                defaultMessage: 'No recently viewed items',
-              })}
-            </p>
-          </EuiText>
+        {/* Workspace name and Overview inside workspace */}
+        {currentWorkspace && (
+          <>
+            <EuiCollapsibleNavGroup iconType={'folderClosed'} title={currentWorkspace.name} />
+            <EuiCollapsibleNavGroup
+              onClick={async () => {
+                closeNav();
+                await navigateToApp(WORKSPACE_APP_ID, {
+                  path:
+                    PATHS.update +
+                    '?' +
+                    WORKSPACE_ID_IN_SESSION_STORAGE +
+                    '=' +
+                    currentWorkspace.id,
+                });
```

Comment:
can we reuse method `getWorkspaceUrl` and do navigation by `navigateToUrl`?

Reply:
Ok, I will reuse getWorkspaceUrl